usaspending_api/agency/tests/integration/test_awards.py

```diff
@@ -0,0 +1,175 @@
+import pytest
+
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.awards.models import TransactionNormalized
+from usaspending_api.search.tests.data.utilities import setup_elasticsearch_test
+
+
+@pytest.fixture
+def transaction_search_1():
+
+    # Submission
+    dsws = mommy.make(
+        "submissions.DABSSubmissionWindowSchedule",
+        submission_reveal_date="2021-04-09",
+        submission_fiscal_year=2021,
+        submission_fiscal_month=7,
+        submission_fiscal_quarter=3,
+        is_quarter=False,
+        period_start_date="2021-03-01",
+        period_end_date="2021-04-01",
+    )
+    mommy.make("submissions.SubmissionAttributes", toptier_code="001", submission_window=dsws)
+    mommy.make("submissions.SubmissionAttributes", toptier_code="002", submission_window=dsws)
+
+    # Toptier and Awarding Agency
+    toptier_agency_1 = mommy.make(
+        "references.ToptierAgency",
+        toptier_code="001",
+    )
+
+    toptier_agency_2 = mommy.make(
+        "references.ToptierAgency",
+        toptier_code="002",
+    )
+
+    awarding_agency_1 = mommy.make("references.Agency", toptier_agency=toptier_agency_1, toptier_flag=True)
+    awarding_agency_2 = mommy.make("references.Agency", toptier_agency=toptier_agency_2, toptier_flag=True)
+
+    # Awards
+    award_contract = mommy.make("awards.Award", category="contract")
```

Comment:
Shouldn't these have the `type` set? Also, can you add a test for the award types.

Reply:
I guess this one doesn't matter since Transaction Index pulls type from the TransactionNormalized record.